code/game/world.dm

```diff
@@ -357,6 +344,10 @@ GLOBAL_VAR(restart_counter)
 	if (features)
 		s += ": [jointext(features, ", ")]"
 
+	s += "<br>Time: <b>[gameTimestamp("hh:mm")]</b>"
+	s += "<br>Alert: <b>[capitalize(get_security_level())]</b>"
+	s += "<br>Roleplay: <b>Medium</b>
```

Comment:
There are multiple reasons why we can't have this:
 - Hardcoding values that downstreams don't follow
 - We don't advertise as a medium roleplay server, just a roleplay server.

Reply:
alright will remove roleplay line